SetValueCurve-exceptions

```diff
@@ -0,0 +1,209 @@
+ 0:01.34 INFO Using 12 client processes
+ 0:06.39 INFO Starting http server on 127.0.0.1:8001
+ 0:06.39 INFO Starting http server on 127.0.0.1:8000
+ 0:06.40 INFO Starting https server on 127.0.0.1:8443
+ 0:08.11 SUITE_START: web-platform-test - running 1 tests
```

Comment:
This file shouldn't be there

Reply:
I think I made a mistake while squashing commits. I will try to where I went wrong

Reply:
This file still needs to be removed

Reply:
Done.

Reply:
Hi Manish, 
Are any other changes needed for this?